components/brave_vpn/resources/panel/components/error-panel/style.ts

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components'
+
+export const Box = styled.div`
+  width: 100%;
+  height: 100%;
+  max-height: 450px;
+  background: ${(p) => p.theme.color.panelBackground};
+  position: relative;
+  overflow: hidden;
+`
+
+export const PanelContent = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 48px 24px 16px 24px;
+  position: relative;
+  z-index: 2;
+`
+
+export const ReasonTitle = styled.h1`
+  color: ${(p) => p.theme.color.text01};
+  font-family: ${(p) => p.theme.fontFamily.heading};
+  font-size: 18px;
+  font-weight: 600;
+  letter-spacing: 0.02em;
+  line-height: 2.6;
+  margin: 0;
+  text-align: center;
+`
+
+export const ReasonDesc = styled.p`
+  color: ${(p) => p.theme.color.text01};
+  font-family: ${(p) => p.theme.fontFamily.heading};
+  font-size: 14px;
+  font-weight: 400;
+  letter-spacing: 0.02em;
+  margin: 0;
+  text-align: center;
+  padding-bottom: 24px;
+`
+
+export const IconBox = styled.div`
+  width: 62px;
+  height: 62px;
+
+  svg>path {
+    fill: ${(p) => p.theme.color.text03};
```

Comment:
nit: not ideal IMO as this component comes from brave-ui, so we're assuming an implementation detail that may not apply when this icon gets updated. But, if this can't be accomplished via a prop or a simple `color` attribute, then so be it.